fighthealthinsurance/ml/ml_models.py

```
@@ -2153,47 +2204,85 @@ async def __infer(
-                                f"{response_body_preview}"
-                            )
-                        raise
-                    json_result = await response.json()
-                    if json_result.get("object") == "error":
-                        logger.warning(f"Bad response from {self} with {model}")
+                            response.raise_for_status()
+                        except aiohttp.ClientResponseError as e:
+                            response_body = ""
+                            try:
+                                response_body = await response.text()
+                            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)